executor/seqtest/seq_executor_test.go

```diff
@@ -967,7 +967,7 @@ func (s *seqTestSuite) TestBatchInsertDelete(c *C) {
 		atomic.StoreUint64(&kv.TxnTotalSizeLimit, originLimit)
 	}()
 	// Set the limitation to a small value, make it easier to reach the limitation.
-	atomic.StoreUint64(&kv.TxnTotalSizeLimit, 5000)
+	atomic.StoreUint64(&kv.TxnTotalSizeLimit, 5500)
```

Comment:
This change is because we update the parser, and marshal table info use more bytes (add a new temporary field, now it's 5038 byte).
If I don't change here, CI will fail and complain transaction too large for the `create table ...` statement.